examples/1.0/file-api/database/graphcool.yml

```diff
@@ -6,4 +6,3 @@ cluster: local
 datamodel:
   - datamodel.graphql
   
-secret: ${env:GRAPHCOOL_SECRET}
```

Comment:
why did you remove the secret here?

Reply:
sorry that is not what I had in mind. I will fix that.